py-polars/polars/dataframe/frame.py

```diff
@@ -1285,7 +1285,7 @@ def __dataframe__(
         allow_copy: bool = True,  # noqa: FBT001
     ) -> PolarsDataFrame:
         """
-        Convert to a dataframe object implementing the dataframe interchange protocol.
+        Convert to a DataFrame object implementing the dataframe interchange protocol.
```

Comment:
This should remain "dataframe" as it's not referring to the Polars DataFrame class, it's referring to a generic dataframe object.